tools/mc-java-protoc/src/main/java/io/spine/tools/mc/java/protoc/message/ImplementInterface.java

```diff
@@ -30,20 +30,18 @@
 import io.spine.code.java.ClassName;
 import io.spine.tools.mc.java.protoc.CodeGenerationTask;
 import io.spine.tools.mc.java.protoc.CompilerOutput;
+import io.spine.tools.protoc.JavaClassName;
 import io.spine.type.MessageType;
 
-import static io.spine.util.Preconditions2.checkNotEmptyOrBlank;
-
 /**
  * An abstract base for the interface code generation tasks.
  */
 public abstract class ImplementInterface implements CodeGenerationTask {
```

Comment:
Do we expect it to have custom or user-defined descendants? If so, I would describe that in the documentation and mark it `@SPI`. 

However, to me, this class should safely become package-private for now.